src/material/core/style/_sass-utils.scss

```diff
@@ -0,0 +1,20 @@
+@use 'sass:list';
```

Comment:
Could we make more specific files than a `_utils.scss`? I'm perpetually on a crusade against files named "utils", "helpers", etc. 

Maybe `_token-maps.scss` and `_coercion.scss` ?